accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -599,18 +599,37 @@ non-breaking.
 
 Let `n` be the language version this proposal ships in. Then:
 
+*   `base`, `interface`, `final`, and `mixin` can only be applied to classes and
+    mixins in libraries whose language version is `>= n`.
+
+*   When the `base`, `interface`, `final`, `mixin`, or `sealed` modifiers are
```

Comment:
I think we're still discussing whether we want to carve out an exception for the SDK core libraries.  I'm fine with landing this now though if you want - I don't expect us to change this for non-SDK core libraries.

Reply:
Added a TODO so that if an implementer reads the proposal they can see this might get tweaked.